app/controllers/api/integrations_controller.rb

```diff
@@ -266,10 +260,6 @@ def deploying_hosted_proxy_key
     "#{current_account.id}/deploying_hosted"
   end
 
-  def update_onboarding_mapping_bubble
-    onboarding.bubble_update('mapping') if proxy_rules_added_for_last_method_metric?
-  end
-
   def proxy_rules_added_for_last_method_metric?
```

Comment:
Now this method is not used at all, is it? If that's the case, we should remove it 😄